testing/run-n-times-flaky-test.sh

```diff
@@ -3,7 +3,7 @@
 # This script runs a piglit test n times, and returns how many runs
```

Comment:
Now that the test is more complete, is "runs a piglit test n times" correct? Shouldn't be "runs a piglit or deqp test n times"?